.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - 1.x
+  - 1.7.x
+  - master
+
+install: make get-deps
+script: make build
```

Comment:
Do we really need to build binaries for all platforms, or does `go test` suffice?